rosdep/python.yaml

```diff
@@ -2090,6 +2090,9 @@ python-impacket:
   debian: [python-impacket]
   fedora: [python-impacket]
   ubuntu: [python-impacket]
+python-influxdb:
+  debian: [python-influxdb]
```

Comment:
Oddly, this is available in jessie and buster, but not in the release between them (stretch).  Since we still support stretch, I think we need to call it out separately with a 'null' entry.

Reply:
It's not missing stretch has been delisted from the website I guess they're pulling down stretch as it's been replaced by buster . The package is actually available in stretch.


<details>

```
tfoote@snowman4:~ Last: [0] (0s Seconds)
$ rocker debian:stretch 
Plugins found: ['dev_helpers', 'env', 'git', 'home', 'nvidia', 'pulse', 'ssh', 'user', 'x11']
Active extensions []
Writing dockerfile to /tmp/tmpbbwhdvie/Dockerfile
vvvvvv

FROM debian:stretch

^^^^^^
Building docker file with arguments:  {'path': '/tmp/tmpbbwhdvie', 'rm': True, 'nocache': False}
building > Step 1/1 : FROM debian:stretch
building >  ---> bd753a4a335e
building > Successfully built bd753a4a335e
Executing command: 
docker run -it   --rm      bd753a4a335e 
root@44284de1d705:/# apt-get update
Ign:1 http://deb.debian.org/debian stretch InRelease
Get:2 http://security.debian.org/debian-security stretch/updates InRelease [94.3 kB]
Get:3 http://deb.debian.org/debian stretch-updates InRelease [91.0 kB]
Get:4 http://deb.debian.org/debian stretch Release [118 kB]         
Get:5 http://deb.debian.org/debian stretch Release.gpg [2410 B]
Get:6 http://security.debian.org/debian-security stretch/updates/main amd64 Packages [525 kB]
Get:7 http://deb.debian.org/debian stretch-updates/main amd64 Packages [27.9 kB]
Get:8 http://deb.debian.org/debian stretch/main amd64 Packages [7083 kB]       
Fetched 7942 kB in 1s (4846 kB/s)  
Reading package lists... Done
root@44284de1d705:/# apt-cache search influxdb
golang-github-influxdb-enterprise-client-dev - Golang client for speaking to the InfluxDB Enterprise application
golang-github-influxdb-usage-client-dev - library for speaking to the InfluxDB Anonymous Usage Reporting API
golang-github-influxdb-influxdb-dev - Scalable datastore for metrics, events, and real-time analytics. Dev package
influxdb - Scalable datastore for metrics, events, and real-time analytics
influxdb-client - command line interface for InfluxDB
influxdb-dev - Transitional package for golang-github-influxdb-influxdb-dev
ruby-influxdb - library for InfluxDB
```
</details>

Reply:
```
root@44284de1d705:/# apt-cache search influxdb
golang-github-influxdb-enterprise-client-dev - Golang client for speaking to the InfluxDB Enterprise application
golang-github-influxdb-usage-client-dev - library for speaking to the InfluxDB Anonymous Usage Reporting API
golang-github-influxdb-influxdb-dev - Scalable datastore for metrics, events, and real-time analytics. Dev package
influxdb - Scalable datastore for metrics, events, and real-time analytics
influxdb-client - command line interface for InfluxDB
influxdb-dev - Transitional package for golang-github-influxdb-influxdb-dev
ruby-influxdb - library for InfluxDB
```

It looks that the output of `apt-cache search influxdb` does not include python-influxdb. I will remote the package for stretch distribution.